.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
How about just installing `llvm-12` at line 34?

Reply:
You may replace `llvm-12` with `clang-12` as recommended in the [README](https://github.com/flutter-tizen/embedder#prerequisites). It seems `clang` depends on `llvm` but not vice versa.

Reply:
Thank you for your advice. I updated PR